setting-up/migrate.md

```diff
@@ -0,0 +1,125 @@
+* The `gzip` file will be copied to the new image's mounting folder in the very next step to be imported into the new container's internal `MongoDB` service
+
+```
+# Create backup folder to store dump file
+docker exec appsmith_mongo_1 bash -c 'mkdir -p /data/db/backup'
+
+# Dumping Mongo data and compressed into a gzip file
+docker exec appsmith_mongo_1 bash -c 'mongodump --uri=$APPSMITH_MONGODB_URI --archive=/data/db/backup/appsmith-data.archive --gzip'
+```
+
+*Note: The container name may be different in case you have made changes on the `docker-compose.yml` file. Please check and use the correct `MongoDB` container name*
+
+### 2. Migrate Configuration
+* Firstly, we use `docker.env` file to store all environment variables (include variables from both `encryption.env` and current `docker.env` files)
+* The below set of commands will help you to copy all current variables into a new `docker.env` file which will be stored in the mounting folder of the new Fat container image
+
+```
+# You should run these commands in the parent folder of appsmith-docker-stack and appsmith-fat-container
+# Create appsmith-fat-container folder
+mkdir -p appsmith-fat-container
+
+# Create mounting volume folder
+mkdir -p appsmith-fat-container/stacks
+
+# Export variable to store the path to old and new folders
+export OLD_APPLICATION_DIR=$PWD/appsmith-docker-stack/appsmith
+export NEW_APPLICATION_DIR=$PWD/appsmith-fat-container
+
+# Create configuration folder
+mkdir -p $NEW_APPLICATION_DIR/stacks/configuration
+
+# Copy environment variable from docker.env & encryption.env 
+cat $OLD_APPLICATION_DIR/docker.env $OLD_APPLICATION_DIR/encryption.env >> $NEW_APPLICATION_DIR/stacks/configuration/docker.env
+```
+* We also introduce a new application stack in Appsmith system - `Realtime Service (RTS)`. Please run the below command to configure the backend URL for `RTS`
+```
+# Add backend URL configuration
+echo APPSMITH_API_BASE_URL=http://localhost:8080 >> $NEW_APPLICATION_DIR/stacks/configuration/docker.env
+```
+
+*Note: In our new image, we have used Redis and MongoDB service inside the container as default services. Therefore, if you do not use external services for Redis and MongoDB, you will need to change 2 environment variables in the `docker.env` file (Replace `<MONGO_USERNAME>` and `<MONGO_PASSWORD>` with existing `MONGO_INITDB_ROOT_USERNAME`  and `MONGO_INITDB_ROOT_PASSWORD`  in the `docker.env` file)*
+```
+APPSMITH_MONGODB_URI=mongodb://<MONGO_USERNAME>:<MONGO_PASSWORD>@localhost/appsmith
+APPSMITH_REDIS_URL=redis://127.0.0.1:6379
+```
+### 3. Export https config & certificate (Optional)
+
+* If you have deployed Appsmith with custom domain, you might want to move your certificate to the new container
+
+```
+# Add custom domain to configuration file
+echo APPSMITH_CUSTOM_DOMAIN=<YOUR-DOMAIN> >> $NEW_APPLICATION_DIR/stacks/configuration/docker.env
```

Comment:
If the user already has a custom domain configured, like the above paragraph says, then the `docker.env` would already have this env variable set right? So do we still need this command to be run here?